nncf/openvino/quantization/helpers.py

```diff
@@ -66,13 +67,42 @@ def _convert_compressed_model_to_openvino_model(model: pot.graph.nx_model.Compre
     return ov_model
 
 
+def _create_ignored_scope_config(ignored_scope: IgnoredScope) -> Dict:
+    """
+    Maps the content of `IgnoredScope` class to the `ignored` section of POT config.
+
+    :param ignored_scope: The ignored scope
+    :return: A POT ignored scope configuration as dict
+    """
+    if ignored_scope is None:
+        return {}
+
+    ignored = {}
+    if ignored_scope.names is not None:
+        names = ignored_scope.names
+        if isinstance(names, str):
+            names = [names]
+        ignored['scope'] = names
+    if ignored_scope.patterns is not None:
+        raise RuntimeError('Quantization algorithm form the OpenVINO backend '
+                           'does not support regular expressions in the ignored '
+                           'scopes yet')
+    if ignored_scope.types is not None:
+        types = ignored_scope.types
+        if isinstance(types, str):
+            types = [types]
```

Comment:
The same as above.

Reply:
It's just sugar... avoids common mistakes when defining an ignored scope